addon/stories/pix-tooltip.stories.js

```diff
@@ -6,7 +6,9 @@ export const tooltip = (args) => {
       <PixTooltip
         @text={{text}}
         @position={{position}}
+        @light={{light}}
         @inline={{inline}}
+        @wide={{wide}}
```

Comment:
**suggestion:** rendre les booléens plus explicite
```suggestion
        @isLight={{isLight}}
        @isInline={{isInline}}
        @isWide={{isWide}}
```

Reply:
le inline existait déjà, comment on procède pour faire évoluer dans les app qui l'utilisaient ?

Reply:
ah.. bonne question!
Peut-être dans un premier temps accepter `isInline` et `inline` en attributs, avec `isInline` qui prend la valeur de `inline` seulement si `isInline` est vide ? Puis supprimer `inline` dans un second temps / autre PR.

C'est-à-dire, dans le pix-tooltip.js avoir:
```
get isInline() {
  return this.args.isInline ? this.args.isInline : this.args.inline;
}
```

et côté .hbs, ne plus utiliser `@inline` mais la méthode précédente`this.isInline`.

Qu'en penses-tu ?
C'est un peu compliqué car il faut penser à revenir nettoyer pix-ui après merge et modif côté apps front... :S

Reply:
Sinon autre solution plus fastidieuse : faire une release majeure de Pix UI. Puis pour éviter les erreurs lors de mises à jour faire une PR dans la foulée qui corrige les `inline` en `isInline`.
De ce que je vois il y a : **PixAdmin** qui est impacté (les tooltip sur le menu), **mon-pix** pour le code de vérification d'une certif sur un certificat et **PixOrga**.
Après en relativisant, si le inline se perds sur ces app, il me semble que l'affichage sera dégradé mais ça permettra toujours de lire le contenu de la tooltip. Je pense qu'il faut pas se priver de faire des releases majeures ... et au pire corriger plus tard si besoin est.

Reply:
j'ai opté pour la montée de version majeure !
Je propose une nomenclature pour identifer les PR nécéssitant une montée de version majeure (cf le titre de cette PR : `[TYPE] <Titre> - BREAKING CHANGES (Pix-XXXX).`)